### PR TITLE
Release/website fixes

### LINKED ISSUE #392: Automatically cross-publish sbt plugin
`sbt-scalafix` cross-builds correctly but the CI only publishes the 0.13 version.

### PR DESCRIPTION
**Pull request by olafurpg**
- Fixes #392

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -17,12 +17,12 @@ name := {
...
 lazy val crossVersions = Seq(scala211, scala212)
 
-// Custom scalafix release command. Tried sbt-release but didn't play well with sbt-doge.
-commands += Command.command("release") { s =>
+commands += Command.command("ci-release") { s =>
   "clean" ::
     "very publishSigned" ::
-    "sonatypeRelease" ::
-    "gitPushTag" ::
+    "^^ 1.0.2 " ::
```
Comment (gabro) [inline build.sbt]: doesn't this publish only 1.0.2? Shouldn't it be `^ scalafix-sbt/publishSigned :: sonatypeReleaseAll`?

Comment (olafurpg) [reply build.sbt]: `very publishSigned` publishes the 0.13 version

Comment (gabro) [reply build.sbt]: Ah ok, I'm still fairly surprised that ^ publishSigned doesn't do what I thought :(

Comment (olafurpg) [reply build.sbt]: The way I understand it, `^` will aggregate `sbtCrossVersions` like `very` does for `crossScalaVersions`. I haven't found a way to combine the two with `^ very publishSigned`, this was the easiest solution I could think of.

**Pull request merged**